dom0/sd-dom0-files.sls

```diff
@@ -38,7 +38,7 @@ dom0-workstation-rpm-repo:
         gpgcheck=1
         gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test
         enabled=1
-        baseurl=https://dev-bin.ops.securedrop.org/dom0-rpm-repo/
+        baseurl=https://yum.securedrop.org/workstation/dom0/f25
```

Comment:
ah yeah you're right we should make this `yum-test` so it's clear this is not for production use

Reply:
Roger, agreed: I'll update the URL here and re-ping for review.

Reply:
Updated, ready for full review.

Reply:
I wish for it to be `rpm.securedrop.org` as yum is already obsolete.

Reply:
> I wish for it to be `rpm.securedrop.org` as yum is already obsolete.

The only reason I chose "yum{,-test}" is that's what the Qubes repo use: https://yum.qubes-os.org/ Since we're still in dev, we can change the URL again in the near future, but it'll take some coordination with the backend.